File: src/main/scala/com/bombproj/vo/ProjectDatabaseVO.java
package com.bombproj.vo;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import java.util.Date;

@Data
public class ProjectDatabaseVO {

    private Integer id;
    private String databaseName;
    private String databaseDesc;
    private String dbHost;
    private String dbPort;
    private String userName;
    private String version;
    private String type;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm")
    private Date updateTime;
    private Date createTime;
    private String createUser;

}

File: src/main/scala/com/bombproj/vo/ProjectDatabaseVO.scala
package com.bombproj.vo

import java.util.Date

import com.fasterxml.jackson.annotation.JsonFormat

import scala.beans.BeanProperty

case class ProjectDatabaseVO() {

    @BeanProperty var id: Int = _
    @BeanProperty var databaseName: String = _
    @BeanProperty var databaseDesc: String = _
    @BeanProperty var dbHost: String = _
    @BeanProperty var dbPort: String = _
    @BeanProperty var userName: String = _
    @BeanProperty var version: String = _
    @BeanProperty var `type`: String = _
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm") private var updateTime: Date = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var createUser: String = _

}
